pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```
@@ -29,20 +26,19 @@
 , ...
 } @ args:
 
-assert (pname != null || version != null) -> (name == null && pname != null); # You must declare either a name or pname + version (preferred).
+assert (!args ? pname || !args ? version) -> (args ? name); # You must provide name if pname or version (preferred) is missing.
```

Nit:

```suggestion
assert (!args ? pname || !args ? version) -> (args ? name); # You must provide name if pname and version (preferred) are missing.
```

It's not an *or*, you must have pname *and* version.

---

the expression `!args ? pname || !args ? version` (equivalent to `!(args ? pname && args ? version)`) will be false if either pname or version is missing, so the comment as-is seems accurate to me.